test/consensus_cogateway/withdraw.js

```diff
@@ -0,0 +1,124 @@
+// Copyright 2020 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+const BN = require('bn.js');
+
+const MockToken = artifacts.require('MockToken');
+const ConsensusCogateway = artifacts.require('ConsensusCogateway');
+const MockConsensus = artifacts.require('MockConsensus');
+
+const { AccountProvider } = require('../test_lib/utils.js');
+const Utils = require('../test_lib/utils.js');
+
+contract('ConsensusCogateway::withdraw', (accounts) => {
+  const accountProvider = new AccountProvider(accounts);
+  let param;
+  let utmost;
+
+  let consensusCogateway;
+  beforeEach(async () => {
+    consensusCogateway = await ConsensusCogateway.new();
+    const owner = accountProvider.get();
+    utmost = await MockToken.new(18, { from: owner });
+    param = {
+      metachainId: Utils.getRandomHash(),
+      utmostAddress: utmost.address,
+      owner,
```

Comment:
```suggestion
      owner: accountProvider.get(),
```
Remove `const owner = accountProvider.get();` from line 34.

Reply:
Done